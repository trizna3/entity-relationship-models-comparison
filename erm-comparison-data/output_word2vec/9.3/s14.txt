rooms-rooms
payments-payments
contracts-contracts
buildings-buildings;floors
account;students-students;account
REBIND_MN_TO_1NN1:S-contracts-account
CHANGE_CARDINALITY:S-buildings-floors
CHANGE_CARDINALITY:S-floors-buildings
CONTRACT_11_ASSOCIATION:S-buildings-floors
CONTRACT_11_ASSOCIATION:S-students-account
