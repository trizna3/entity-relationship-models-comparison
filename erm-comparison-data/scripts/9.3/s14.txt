entity set buildings {
	"address"
};

entity set floors {
};

entity set rooms {
	"fee"
	"capacity"
};

entity set contracts {
	"start"
	"end"
};

entity set students {
	"name"
};

entity set account {
	"credit"
};

association between 
{
	buildings MANY
	floors MANY
}{};

association between 
{
	floors MANY
	rooms MANY
}{};

association between 
{
	rooms ONE 
	students MANY
}{};

association between 
{
	contracts MANY
	students ONE
}{};

association between 
{
	students ONE
	account ONE
}{};

association "payments" between 
{
	contracts MANY
	account	MANY
}{
	"time"
};

