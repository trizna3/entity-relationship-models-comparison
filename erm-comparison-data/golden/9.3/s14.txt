buildings-buildings
rooms-rooms
contracts-contracts
payments-payments
account-account
students-students
REBIND_MN_TO_1NN1:S-contracts-account